bin/gen-crons.py

```diff
@@ -23,6 +23,8 @@ def main():
                             'Only define for cron.d style crontabs.'))
     parser.add_option('-p', '--python', default='python2.6',
                       help='Python interpreter to use.')
+    parser.add_option('-n', '--nologdir', action='store_true',
```

Comment:
HA! global.conf. I'm clearly not thinking straight today.

So then we'd have to file a bug to get Ops to add the log dir to chief settings... meh. Doesn't seem that big a deal.  This is all because we want the app to ensure a directory we already know exists exists. Let's not file a bug for it.